src/test/kotlin/com/terraformation/backend/seedbank/model/accession/Helpers.kt

```diff
@@ -0,0 +1,172 @@
+package com.terraformation.backend.seedbank.model.accession
+
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.db.seedbank.AccessionId
+import com.terraformation.backend.db.seedbank.AccessionState
+import com.terraformation.backend.db.seedbank.DataSource
+import com.terraformation.backend.db.seedbank.ProcessingMethod
+import com.terraformation.backend.db.seedbank.SeedQuantityUnits
+import com.terraformation.backend.db.seedbank.ViabilityTestId
+import com.terraformation.backend.db.seedbank.ViabilityTestResultId
+import com.terraformation.backend.db.seedbank.ViabilityTestSubstrate
+import com.terraformation.backend.db.seedbank.ViabilityTestType
+import com.terraformation.backend.db.seedbank.WithdrawalId
+import com.terraformation.backend.db.seedbank.WithdrawalPurpose
+import com.terraformation.backend.seedbank.grams
+import com.terraformation.backend.seedbank.model.AccessionModel
+import com.terraformation.backend.seedbank.model.SeedQuantityModel
+import com.terraformation.backend.seedbank.model.ViabilityTestModel
+import com.terraformation.backend.seedbank.model.ViabilityTestResultModel
+import com.terraformation.backend.seedbank.model.WithdrawalModel
+import com.terraformation.backend.seedbank.seeds
+import java.time.Clock
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneOffset
+
+val today = january(2)
+val todayInstant = today.atStartOfDay(ZoneOffset.UTC).toInstant()
+val clock: Clock = Clock.fixed(todayInstant, ZoneOffset.UTC)
+val tomorrow = today.plusDays(1)
+val tomorrowInstant = tomorrow.atStartOfDay(ZoneOffset.UTC).toInstant()
+val tomorrowClock = Clock.fixed(tomorrowInstant, ZoneOffset.UTC)
+val yesterday = today.minusDays(1)
+val yesterdayInstant = yesterday.atStartOfDay(ZoneOffset.UTC).toInstant()
+val yesterdayClock = Clock.fixed(yesterdayInstant, ZoneOffset.UTC)
+
+var viabilityTestResultId = ViabilityTestResultId(1)
+var viabilityTestId = ViabilityTestId(1)
+var withdrawalId = WithdrawalId(1)
+var defaultState = AccessionState.Processing
```

Comment:
Moving these to top-level scope is going to be problematic, I think. First, they'll become singletons that are shared across tests. So if, for example, a test sets a new value for `viabilityTestId`, the next test will also see that new value because the initializer here only runs once rather than once per test method. That could cause tests to fail depending on what order they're run. Related to that, these objects will all stay around in memory for the entire rest of the run of the test suite once they're referenced. They're small enough that that won't matter much in practice but it still means less test isolation.

The other, probably bigger, concern I have is that it's going to be too easy to use these by mistake. Right now, if I type `clock` in a test method body and I've forgotten to declare a `clock` in my test class, it's an error. With this change, IntelliJ might auto-import it from here, with me none the wiser. I chose `clock` as an example because in a lot of tests, we use a MockK stub instead of `Clock.fixed()` to allow tests to manipulate the time, so the static import would cause runtime failures.

What do you think about putting these variables/functions in an abstract superclass instead?

Reply:
Oh, very good point. I like the idea of an abstract superclass! I'll make that change.